research-software-directory/RSD-as-a-service
# Add support for website tracking via Matomo

# Add support for website tracking via Matomo

Fixes #435

Changes proposed in this pull request:

* similar to Hotjar: now Matomo is also supported for website tracking

How to test:

* set the env variables MATOMO_URL and MATOMO_ID
* you should then see a request from the frontend to `<YOUR_MATOMO_URL>/piwik.php?...&idsite=<YOUR_ID>`
* if one (or both) of the variables is not set no request to matomo is send

PR Checklist:
